README.aspnetcore-build.md

```diff
@@ -19,7 +23,7 @@ This repository contains images that are used to compile/publish ASP.NET Core ap
 - [`1.1.8-1.1.9-nanoserver-sac2016`, `1.1.8-1.1.9`, `1.1`, `1` (*1.1/nanoserver-sac2016/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/1.1/nanoserver-sac2016/sdk/Dockerfile)
 - [`2.0.8-2.1.200-nanoserver-sac2016`, `2.0-nanoserver-sac2016`, `2.0.8-2.1.200`, `2.0`, `2`, `latest` (*2.0/nanoserver-sac2016/sdk/Dockerfile*)](https://github.com/aspnet/aspnet-docker/blob/master/2.0/nanoserver-sac2016/sdk/Dockerfile)
 
->**Note:** ASP.NET Core multi-arch tags, such as 2.0, have been updated to use nanoserver-1709 images if your host is Windows Server 2016 Version 1709 or higher or Windows 10 Fall Creators Update (Version 1709) or higher. You need Docker 17.10 or later to take advantage of these updated tags.
+>**Note:** ASP.NET Core multi-arch tags, such as 2.0, have been updated to use nanoserver-1803 images if your host is Windows Server 2016 Version 1803 or higher or Windows 10 Fall Creators Update (Version 1803) or higher. You need Docker 17.10 or later to take advantage of these updated tags.
```

Comment:
This note is kind off odd now because it still applies to 1709.  We dropped this from microsoft/dotnet.

Reply:
I'll remove it